### PR TITLE
optimized CSS / added responsive QuizPage

### PR DESCRIPTION
**Pull request by JulienOza**

### CODE REVIEW

**Review by Rayane0823 — APPROVED** [review]

File: frontend/src/components/Clues/Clues.jsx [hunk 1]
```diff
@@ -3,12 +3,12 @@ import "./Clues.css";
 
 function Clues() {
   return (
-    <>
-      <p className="Clues Date-clue">1993</p>
-      <p className="Clues Producer-clue">Steven Spielberg</p>
-      <p className="Clues Genre-clue">Science-fiction / Aventure</p>
-      <p className="Clues Casting-clue">Sam Neill / Laura Dern</p>
-    </>
+    <section className="Clues">
```
Comment (Yoann-Fortin) [inline frontend/src/components/Clues/Clues.jsx]: No PascalCase in CSS

File: frontend/src/components/Homepage/Homepage.css [hunk 2]
```diff
@@ -45,94 +23,56 @@ body {
...
 
-img {
+.logo-home {
+  margin-top: 1rem;
   width: auto;
-  height: 15rem;
+  height: 30vh;
+}
+
+.OpenedPopUp {
```
Comment (Yoann-Fortin) [inline frontend/src/components/Homepage/Homepage.css:60]: No PascalCase

File: frontend/src/components/NamePopUp/NamePopUp.css [hunk 3]
```diff
@@ -4,65 +4,93 @@
...
   text-decoration: none;
   padding-top: 0.9rem;
   padding-bottom: 0.9rem;
   margin-top: 0.5rem;
-  width: 50%;
+  text-align: center;
+  width: 100%;
+}
+
+.PopUpButton:hover {
```
Comment (Yoann-Fortin) [inline frontend/src/components/NamePopUp/NamePopUp.css]: No PascalCase

File: frontend/src/components/Navbar/Navbar.css [hunk 4]
```diff
@@ -1,28 +1,10 @@
...
-  padding: 0.2rem;
+.navbar_container {
   width: 90%;
-}
-
-.nav__button {
+  height: auto;
   display: flex;
-  justify-content: center;
+  flex-direction: row;
```
Comment (Yoann-Fortin) [inline frontend/src/components/Navbar/Navbar.css]: Dead code

File: frontend/src/components/quiz-page/QuizPage.css [hunk 5]
```diff
@@ -1,19 +1,54 @@
...
+
+@media screen and (min-width: 480px) {
+  .quizpage_container {
+    display: grid;
+    grid-template-columns: 60% 40%;
+    grid-template-rows: 10% 80% 10%;
+    justify-items: center;
+  }
+
+  .Clues {
```
Comment (Yoann-Fortin) [inline frontend/src/components/quiz-page/QuizPage.css]: No PascalCase

**Review by Yoann-Fortin — CHANGES_REQUESTED** [review]

File: frontend/src/components/quiz-page/QuizPage.css (hunk 5 above)
Comment (JulienOza) [reply frontend/src/components/quiz-page/QuizPage.css]: Ok, changed

File: frontend/src/components/Navbar/Navbar.css (hunk 4 above)
Comment (JulienOza) [reply frontend/src/components/Navbar/Navbar.css]: OK removed

File: frontend/src/components/NamePopUp/NamePopUp.css (hunk 3 above)
Comment (JulienOza) [reply frontend/src/components/NamePopUp/NamePopUp.css]: Ok, changed

File: frontend/src/components/Homepage/Homepage.css (hunk 2 above)
Comment (JulienOza) [reply frontend/src/components/Homepage/Homepage.css:60]: Ok, changed

File: frontend/src/components/Clues/Clues.jsx (hunk 1 above)
Comment (JulienOza) [reply frontend/src/components/Clues/Clues.jsx]: Ok, changed

**Review by LucasMerino10 — APPROVED** [review]

**Review by SebSamPro — APPROVED** [review]

**Review by Yoann-Fortin — APPROVED** [review]

**Pull request merged**